spk/htpcmanager/src/installer.sh

```diff
@@ -30,6 +38,9 @@ postinst ()
     # Create a Python virtualenv
     ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
 
+    # Clone the repository
+    ${GIT} clone -q -b ${wizard_fork_branch:=master} ${wizard_fork_url:=git://github.com/styxit/HTPC-Manager.git} ${INSTALL_DIR}/var/HTPC-Manager
```

Comment:
Imo, if the software can update itself, it makes sense to put this inside an `if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ];` statement. The restore during postupgrade would replace the newly-cloned code, so it doesn't do anything worthwhile, and it slows down the upgrade process because repos are cloned with each upgrade.

Sidenote: We do the same with at least some of the other -custom packages (maybe all of them?), so we'll want to use the same approach for those too.